axon-framework/modules.md

```diff
@@ -83,18 +83,18 @@ Besides main modules, there are several extension modules which complement Axon
 
 | Module | Artifact Id | Group Id | Maven Central |
```

Comment:
Might be interesting to move this to the extensions module eventually.

Reply:
Agreed.